src/main.js

```diff
@@ -55,6 +59,26 @@ const filterBeforeCreate = (toast, toasts) => {
 
 const app = createApp(App).use(store).use(router);
 
+if (
+  ["staging", "production"].includes(process.env.NODE_ENV) &&
+  process.env.VUE_APP_SENTRY_DSN
+) {
+  // Since Vue3 isn't officially supported yet by Sentry, we're using
+  // the JavaScript Sentry integration with @sentry/browser.
+  // Refer Sentry documentation for more configs: https://docs.sentry.io/platforms/javascript/configuration/
+  Sentry.init({
+    dsn: process.env.VUE_APP_SENTRY_DSN,
+    integrations: [new Integrations.BrowserTracing()],
+    // Set tracesSampleRate to 1.0 to capture 100%
+    // of transactions for performance monitoring.
+    tracesSampleRate: 1.0,
+    environment: process.env.NODE_ENV,
+  });
+  app.config.errorHandler = (err) => {
```

Comment:
any way to identify the user here?

Reply:
We can use the practices as mentioned below but have a look at my backend comment first about user privacy.
https://docs.sentry.io/platforms/javascript/data-management/sensitive-data/


Backend comment:
https://github.com/avantifellows/plio-backend/pull/202#discussion_r658587962

Reply:
based on our conclusion on the backend, anything to update here?

Reply:
Updated. I've set the user id now if present in vue store.